q2_types/feature_table/_type.py

```diff
@@ -30,5 +30,10 @@
 
 plugin.register_semantic_type_to_format(
     FeatureTable[Frequency | RelativeFrequency | PresenceAbsence],
-    artifact_format=FeatureTableDirectoryFormat
+    artifact_format=BIOMV210DirFmt
+)
+
+plugin.register_semantic_type_to_format(
```

Comment:
Sorry looks like this got missed, it still needs to be removed. (Artifacts record the format of their data-dir, so it shouldn't be a problem when we update the default registration (such as in this PR), the artifact will still load via whatever `metadata.yml` says and the appropriate transformers will be invoked.)